CONTRIBUTING.md

```diff
@@ -0,0 +1,315 @@
+
+If you would like to contribute to the project yourself directly, it's recommended to [create an issue](https://github.com/umcu/clinlp/issues/new) to discuss your idea beforehand. This way, we can make sure that your contribution is in line with the project's goals and that it is not already being worked on by someone else. Of course, for small changes that only touch a couple of lines of code, you can also directly create a pull request. When you are ready to start working on your contribution, please follow the steps outlined in the [Pull requests](#pull-requests) section.
+
+## Pull requests
+
+If you would like to contribute to the project by making changes to the codebase, please follow these steps:
+
+- Fork the repository
+- Create a new branch for your changes
+- Make your changes locally
+  - Check your changes are in line with the project's [Coding Standards](#coding-standards)
+  - Document your changes in `CHANGELOG.md`
+- Push your changes to your fork
+- Create a pull request
+- Wait for feedback
+- Address the feedback
+
+Normally, the pull request will be merged by maintainers after all feedback has been addressed, after which it will be included in the next release.
+
+## Roadmap
+
+We like to keep the project roadmap open and transparent. You can find the roadmap in the [projects](https://github.com/orgs/umcu/projects/3). All issues are automatically added to the roadmap, so you can see what is being worked on, and what is planned for the future.
+
+## Developer environment setup
+
+You can setup a local development environment by cloning the repository:
+
+```bash
+git clone git@github.com:umcu/clinlp.git
+cd clinlp
+```
+
+We use poetry for managing dependencies, and building the package. If you do not have it yet, installation is covered in the [official Poetry guide](https://python-poetry.org/docs/#installation).
+
+Then, you can install the project with dependencies using:
+
+```bash
+poetry install -e . --group dev 
+```
+
+### Repository structure
+
+The repository is structured as follows:
+
+| Directory    | Description                                               |
+| ------------ | --------------------------------------------------------- |
+| `.github`    | GitHub actions and workflows                              |
+| `docs`       | The documentation                                         |
+| `media`      | Media files for the documentation or readme               |
+| `scripts`    | Scripts for simple tasks that are not part of the package |
+| `src/clinlp` | The source code for `clinlp`                              |
+| `tests`      | The tests for `clinlp`                                    |
+
+## Coding standards
+
+With `clinlp` we aim for code that is production-ready, well tested, scalable, maintainable, etc. We have a few guidelines that we follow to ensure that the codebase maintains a high quality.
```

Comment:
Het 'etc' klinkt een beetje achteloos. Ik zou het weghalen, dus er gewoon "With 'clinlp' we aim for code that is production-ready, well tested, scalable and maintainable." van maken.